draft-ietf-quic-transport.md

```diff
@@ -678,6 +687,34 @@ packet protection in detail.  After decryption, the plaintext consists of a
 sequence of frames, as described in {{frames}}.
 
 
+## Compound Packets {#packet-compound}
+
+Compound packets allow the sender to assemble an Initial ({{packet-initial}})
+or a Handshake ({{packet-handshake}}) packet and a Protected
+({{packet-protected}}) packet into one UDP packet, thereby reducing the number
+of packets required to be emitted when application data can be sent during the
+handshake.
+
+A Compound-Initial packet is identical to an Initial packet with the exception
+being that it has the Payload Length field.  The field designates the length of
+the payload of the Initial packet.  The remainder of the UDP packet contains a
+0-RTT Protected packet.
+
+A Compound-Handshake packet is identical to a Handshake packet with the
+exception being that it has the Payload Length field.  The field designates the
+length of the payload of the Handshake packet.  The remainder of the UDP packet
+contains either a 0-RTT Protected packet or a short header packet
+({{short-header}}).
```

Comment:
It might be worth explaining or giving an example that all the packet header fields are there for the subsequent packet.  If that includes the type, by the way, you probably don't actually have to specify what type will be there, since the packet will declare it.